src/types/rxjs.d.ts

```diff
@@ -0,0 +1,296 @@
+export class ArgumentOutOfRangeError {
```

Comment:
Why was this rxjs file copied over?

Reply:
Typesricpt was trying to fix an error where some imports were failing and I guess it created this file to add types. It'll be removed.